shaptools/shell.py

```diff
@@ -187,3 +191,25 @@ def remove_user(user, force=False, root_user=None, root_password=None, remote_ho
         else:
             break
     raise ShellError('error removing user {}'.format(user))
+
+def extract_sapcar_cmd(sar_file, options=None, user=None, password=None, remote_host=None):
```

Comment:
I don't really like to have so many parameters with `None` default value.
Use `kwargs` to bring them. It looks like we might need to include more things in the future, so the optional parameters are better in `kwargs` here.

Reply:
Yes it makes sense to use `kwargs` here